src/_actions/WorkspaceActions.js

```diff
@@ -22,10 +22,15 @@ export const updateTickTradeSubmarket = submarket => ({
     submarket,
 });
 
-export const updateTickTradeDate = date => ({
-    type: types.UPDATE_TRADING_TIMES_DATE,
-    date,
-});
+export const updateTradingTimesDate = date => {
```

Comment:
Did not notice that, just take a look, that action is react on server data and update trading times data, but `UPDATE_TRADING_TIMES_DATE` is to preserve state of selected date, so that it does not jump back to default when user change language.

maybe the name is confusing ?